sdk/shared/src/main/scala/org/ergoplatform/sdk/ReducingInterpreter.scala

```diff
@@ -31,7 +31,7 @@ class ReducingInterpreter(params: BlockchainParameters) extends ErgoLikeInterpre
     * @return data object containing enough data to sign a transaction without Context.
     */
   def reduce(env: ScriptEnv, ergoTree: ErgoTree, context: CTX): ReducedInputData = {
-    val initCost = ergoTree.complexity + context.initCost
```

Comment:
not sure why ergoTree.complexity was there, while in other reduction methods it was removed in 5.0

Reply:
This is by mistake, it seems.